src/metabase/events/sync_database.clj

```diff
@@ -15,31 +14,47 @@
   "Channel for receiving event notifications we want to subscribe to for database sync events."
   (async/chan))
 
+(def ^:private sync-update-channel
+  "Channel for receiving event notifications we want to subscribe to for database sync events."
+  (async/chan))
 
-;;; ## ---------------------------------------- EVENT PROCESSING ----------------------------------------
+(def ^:private sync-delete-channel
+  "Channel for receiving event notifications we want to subscribe to for database sync events."
```

Comment:
This docstring looks like copypasta https://github.com/metabase/metabase/wiki/Metabase-Clojure-Style-Guide#make-sure-to-update-comments